server/migrations/20230510153843-add-integration-to-authentications.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.addColumn(
+        "authentications",
+        "integrationId",
+        {
+          type: Sequelize.UUID,
+          allowNull: true,
+          onDelete: "cascade",
+          unique: true,
+          references: {
+            model: "integrations",
+          },
+        },
+        { transaction }
+      );
+
+      const integrations = await queryInterface.sequelize.query(
+        `SELECT id, "authenticationId" FROM integrations where "authenticationId" is not null`,
```

Comment:
Not sure if this is a good idea for hosted version. Certainly not if there are _too many_ rows.